frontend/src/components/Shared/CrossingMapPage/CrossingMapSearchBar.js

```diff
@@ -1,50 +1,262 @@
 import React, { Component } from 'react';
 import 'components/Shared/CrossingMapPage/CrossingMapSearchBar.css';
 import FontAwesome from 'react-fontawesome';
+import Autosuggest from 'react-autosuggest';
+import MapboxClient from 'mapbox';
+import CrossingMapSearchCrossingSuggestions from 'components/Shared/CrossingMapPage/CrossingMapSearchCrossingSuggestions';
+
+const mapboxClient = new MapboxClient(
+  'pk.eyJ1IjoiY3Jvd2VhdHgiLCJhIjoiY2o1NDFvYmxkMHhkcDMycDF2a3pseDFpZiJ9.UcnizcFDleMpv5Vbv8Rngw',
+);
+
+let autosuggestInput;
+
+// When suggestion is clicked, Autosuggest needs to populate the input
+// based on the clicked suggestion. Teach Autosuggest how to calculate the
+// input value for every given suggestion.
+const getSuggestionValue = suggestion => {
+  return suggestion.place_name || suggestion.name;
+};
+
+// Use your imagination to render suggestions.
+const renderSuggestion = suggestion => (
```

Comment:
I can see how the name `renderSuggestion` matches nicely with the arguments to `Autosuggest`.

But, would it make more sense to name this component `Suggestion`? This would match more closely with how we name other components.  If this component becomes complex enough to justify its own file, we would probably name the file `CrossingMapSearchBar/Suggestion.js`.

Same for `renderSectionTitle`.

Reply:
I dig it